mlrun/feature_store/feature_vector.py

```diff
@@ -502,6 +502,10 @@ def get(self, entity_rows: List[Union[dict, list]], as_list=False):
             if not data:
                 data = None
             else:
+                requested_columns = [
```

Comment:
can you explain why you need changes in get-online?

Reply:
reverted - old bug that I found not relevant for this pr